CI/travis/before_install_darwin.sh

```diff
@@ -1,171 +1,86 @@
-
-cd ${WORKDIR}
-rm master_latest_libiio${LDIST}.pkg*
-wget http://swdownloads.analog.com/cse/travis_builds/master_latest_libiio${LDIST}.pkg
-sudo installer -pkg master_latest_libiio${LDIST}.pkg -target /
-
-cd ${WORKDIR}
-rm -rf libad9361-iio
-if [ ! -d libad9361-iio ]; then
-  git clone https://github.com/analogdevicesinc/libad9361-iio
-  cd libad9361-iio
-  mkdir build
-  cd build
-  cmake ..
-  make
-else
-  cd libad9361-iio
-  git pull
-  cd build
-fi
-sudo make install
-
-cd ${WORKDIR}
-rm -rf gr-iio
-if [ ! -d gr-iio ]; then
-  git clone https://github.com/analogdevicesinc/gr-iio
-  cd gr-iio
-  mkdir build
-  cd build
-  cmake ..
-  make -j4
-else
-  cd gr-iio
-  git pull
-  cd build
-  make -j4
-fi
-sudo make install
-
-brew upgrade python
-
-cd ${WORKDIR}
-rm -rf libsigrok
-if [ ! -d libsigrok ]; then
-  git clone https://github.com/sigrokproject/libsigrok/
-  cd libsigrok
-  ./autogen.sh
-  CC=gcc CXX=g++ CXXFLAGS=-std=c++11 ./configure --disable-static --enable-shared --disable-all-drivers --enable-bindings --enable-cxx
-  make
-else
-  cd libsigrok
-  make >/dev/null
+
+brew_install_or_upgrade() {
+	brew install $1 || \
+		brew upgrade $1 || \
+		brew ls --versions $1 # check if installed last-ly
+}
+
+PACKAGES="qt cmake fftw bison gettext autoconf automake libtool libzip glib libusb python3 python brew-pip"
+PACKAGES="$PACKAGES glibmm doxygen wget boost gnu-sed libmatio dylibbundler libxml2 pkg-config"
+
+for pak in $PACKAGES ; do
+	brew_install_or_upgrade $pak
+done
+
+for pkg in qt gcc bison gettext ; do
+	brew link --overwrite --force $pkg
+done
+
+source ./CI/travis/before_install_lib.sh
+
+QT_PATH="$(brew --prefix qt)/bin"
+export PATH="${QT_PATH}:$PATH"
+
+patch_qwtpolar_mac() {
+	[ -f qwtpolar-qwt-6.1-compat.patch ] || {
+		patch_qwtpolar
+
+		patch -p1 <<-EOF
+--- a/qwtpolarconfig.pri
++++ b/qwtpolarconfig.pri
+@@ -16,7 +16,9 @@ QWT_POLAR_VER_PAT      = 1
+ QWT_POLAR_VERSION      = \$\${QWT_POLAR_VER_MAJ}.\$\${QWT_POLAR_VER_MIN}.\$\${QWT_POLAR_VER_PAT}
+ 
+ unix {
+-    QWT_POLAR_INSTALL_PREFIX    = /usr/local/qwtpolar-\$\$QWT_POLAR_VERSION
++    QWT_POLAR_INSTALL_PREFIX    = $STAGINGDIR
++    QMAKE_CXXFLAGS              = -I${STAGINGDIR}/include
++    QMAKE_LFLAGS                = ${STAGINGDIR}/lib/libqwt*dylib
+ }
+ 
+ win32 {
+EOF
+	}
+}
+
+# Get pip if not installed ; on Travis + OS X, Python is not well supported
+if ! command -v pip ; then
+	curl https://bootstrap.pypa.io/get-pip.py > get-pip.py
+	sudo -H python get-pip.py
 fi
-sudo make install
-
-cd ${WORKDIR}
-rm -rf libsigrokdecode-0.4.1
-if [ ! -d libsigrokdecode-0.4.1 ]; then
-  wget http://sigrok.org/download/source/libsigrokdecode/libsigrokdecode-0.4.1.tar.gz
-  tar -xzvf libsigrokdecode-0.4.1.tar.gz
-  cd libsigrokdecode-0.4.1
-  ./configure
-  make >/dev/null
+
+QMAKE="$(command -v qmake)"
+
+pip install --user cheetah
+
+make_build_wget "libsigrok-0.5.0" "http://sigrok.org/download/source/libsigrok/libsigrok-0.5.0.tar.gz" "" "" "./autogen.sh"
```

Comment:
Could we change this to get the following libsigrok version: https://github.com/sigrokproject/libsigrok

Reply:
ack ; will try ;
i forgot about it :)

thanks